java/compaction/compaction-rust/src/main/java/sleeper/compaction/rust/RustCompactionRunner.java

```diff
@@ -122,6 +122,12 @@ private FFICompactionParams createFFIParams(CompactionJob job, TableProperties t
         params.dict_enc_row_keys.set(tableProperties.getBoolean(DICTIONARY_ENCODING_FOR_ROW_KEY_FIELDS));
         params.dict_enc_sort_keys.set(tableProperties.getBoolean(DICTIONARY_ENCODING_FOR_SORT_KEY_FIELDS));
         params.dict_enc_values.set(tableProperties.getBoolean(DICTIONARY_ENCODING_FOR_VALUE_FIELDS));
+        // Is there a DataFusion iterator set?
+        if (CompactionJob.DATAFUSION_ITERATOR_NAME.equals(job.getIteratorClassName())) {
+            params.iterator_config.set(job.getIteratorConfig());
+        } else {
+            params.iterator_config.set("");
+        }
```

Comment:
We could test this from Java with a basic configuration, I assume as a separate issue.